### PR TITLE
Farzaneh/512 landing page

### PR DESCRIPTION
**Pull request by ghost**
# Description

This UI refers to first page deployement http://localhost:3000/ which is called landing page . In this page I put sign in and sign up button in order to help the user have easy access to this feature. Also, as the first page we announced that all right reserved to our agency. Additionally in future we can add another feature (signup modal and signin modal here.

Fixes #512 (The UI of Landing Page)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

First of all I check the code, and had checked it in a test page after fix the bugs.Then, I added it in index.tsx page in order to use the buttons.I had a few issue about style of this page which I soled it (I should mentioned when you run the project it has a warning "Unexpected any"  which refers to the index.d.ts file I had to make it as a comment then tested again to be sure there is no problem. It worked!)

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by hutan88 — CHANGES_REQUESTED** [review]
Beauty but I think must be didn't  have a scroll

File: frontend/src/pages/index.tsx [hunk 1]
```diff
@@ -1,3 +1,30 @@
 export default function Index() {
-	return <h1>signUp/signIN page</h1>;
+	return (
+		// <!-- This is button of signUp/signIn component -->
+		<div className="bg-white-700 py-9 px-9">
+			<div className="min-h-screen  sm:flex sm:flex-row  justify-center bg-gray-900 p-5 sm:p-60 rounded-3xl shadow-xl">
```
Comment (mhmdrezanaderi) [inline frontend/src/pages/index.tsx:5]: Remove `min-h-screen`
and change  `sm:p-60`  to  `sm:p-10`

Comment (ferizana92) [reply frontend/src/pages/index.tsx]: I changed the style please check again.

**Review by mhmdrezanaderi — APPROVED** [review]
Thanks Farzaneh

**Pull request merged**